### PR TITLE
#129 add `job.ip` column

### PR DESCRIPTION
**Pull request by Suban05**
## Changes
* Added column `ip` 
* `ip` added to `haml` files 
* Tests updated and added

### CODE REVIEW

**Comment by Suban05** [comment]
@yegor256 please, review it

File: liquibase/2024/024-job-ip.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" logicalFilePath="024-ip.xml">
+  <changeSet id="024" author="Suban05">
+    <sql>
+      ALTER TABLE job ADD COLUMN ip INET;
```
Comment (yegor256) [inline liquibase/2024/024-job-ip.xml:28]: @Suban05 I believe, we should make it `NOT NULL` and also update all previous records, setting `0.0.0.0` to it.

**Review by yegor256 — CHANGES_REQUESTED** [review]
@Suban05 looks very good, one small comment above

File: liquibase/2024/024-job-ip.xml (hunk 1 above)
Comment (Suban05) [reply liquibase/2024/024-job-ip.xml:28]: @yegor256 okay

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@Suban05 Thanks for the review! You've earned +5 points for this: +25 as a basis; -40 for reviewing your own contribution (which is a bad idea); +20 to give you at least something.  Your running balance is [+71](https://zerocracy.github.io/judges-action/zerocracy-vitals.html).

**Comment by 0crat** [comment]
@Suban05 Thanks for the contribution! You've earned +23 points for this: +30 as a basis; -7 for too many hits-of-code (201 >= 100). Please, [keep](https://www.yegor256.com/2018/03/06/speed-vs-quality.html) them coming. Your running balance is [+76](https://zerocracy.github.io/judges-action/zerocracy-vitals.html).

**Comment by 0crat** [comment]
@yegor256 Hey there! 👋 Great job on the review! You've snagged 32 points: 25 for being an awesome reviewer, and 7 for those thoughtful comments you made. 💡 Remember, more comments (up to 20) can boost your score even higher next time! Your current balance is [-581](https://zerocracy.github.io/judges-action/zerocracy-vitals.html). Keep up the good work, and you might hit that 80-point max soon! 🚀

**Pull request merged**